packages/rollup-full-node/src/app/test-web3-rpc-handler.ts

```diff
@@ -77,8 +87,12 @@ export class TestWeb3Handler extends DefaultWeb3Handler {
    * Returns the configured timestamp if there is one, else standard timestamp calculation.
    * @returns The timestamp.
    */
-  protected getTimestamp(): number {
-    return super.getTimestamp() + this.timestampIncreaseSeconds
+  protected async getTimestamp(): Promise<number> {
```

Comment:
This should instead return `utils.getTimestamp + this.timestampIncreaseSeconds` and should be used to set the timestamp within sendRawTransaction